### PR TITLE
Add events section on community page

### PR DESCRIPTION
**Pull request by straight-shoota**
grafik

### CODE REVIEW

File: _pages/community.html [hunk 1]
```diff
@@ -257,26 +257,25 @@ <h3>Blogs</h3>
...
-            <span class="location">{{ event.location }}</span>
-          </div>
-        </a>
+<section class="side-section">
+  <h2 id="events">Events</h2>
+  <aside markdown="1">
+  </aside>
+  <div class="cards-list">
+    {% assign events = site.events | reverse %}
+    {% for event in events limit:6 %}
```
Comment (beta-ziliani) [inline _pages/community.html:266]: why the limit?

File: _events/2020-raw-crystal.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+title: Raw Crystal 2020
+start_date: 2020-10-11T16:00
+date_text: 11 December 2020
+redirect_to: https://www.eventbrite.co.uk/e/raw-crystal-2020-tickets-127439094763
```
Comment (beta-ziliani) [inline _events/2020-raw-crystal.md:5]: By adding the youtube link we don't need to create a dedicated event page now. The eventbritte one is totally useless.

```suggestion
redirect_to: https://www.youtube.com/playlist?list=PLfpFq_WLOW__RIIyWvTgYV4bw2cpav2mJ
```

File: assets/events/2023-crystal-berlin.webp
Comment (beta-ziliani) [inline assets/events/2023-crystal-berlin.webp:1]: A good reminder why we need to create the card _before_ the event.

Maybe something like?
image

File: _pages/community.html (hunk 1 above)
Comment (straight-shoota) [reply _pages/community.html:266]: Leftover artifact

**Review by beta-ziliani — APPROVED** [review]

**Pull request merged**